README.md

```diff
@@ -54,32 +54,35 @@ Managed node host:
 * Docker >= 1.12
 * python >= 2.6
 * docker-py >= 1.7.0
+* Java => 8
 
 Make sure that the injector host’s hardware resources is not the performance tests bottleneck. Enough cpu, memory and network resources should be available.
 
-### Common settings
+*Installation Step*:
 
-** HTTP/HTTPS **
-
-By default, the injector will generate HTTPS traffic on the port number 443.
-You can change this setting editing *injector_jmeter_protocol* and *injector_jmeter_target_port* parameters.
+Install the injector Ansible Galaxy roles on the Ansible Control node:
 
+```bash
+ansible-galaxy install hyperfoil.hyperfoil_setup,0.8
+ansible-galaxy install hyperfoil.hyperfoil_shutdown,0.8
+ansible-galaxy install hyperfoil.hyperfoil_test,0.8
+ansible-galaxy install hyperfoil.hyperfoil_generate_report,0.0.1
 ```
-File: roles/injector-configurator/defaults/main.yml
 
-injector_jmeter_protocol: https
-injector_jmeter_target_port: 443
-```
+**Steps**:
 
-### Test your 3scale services
+*Configuration Steps*:
 
-**Steps**:
+**1.** Configure the *ansible_host* parameter by replacing **myinjectorhost.addr.com** with the host IP address/DNS name of the host where you want to install the injector controller component. For example:
 
-**1.** Edit the *ansible_host* parameter by replacing **<injector_host>** with the host IP address/DNS name of the host where you want to install the injector component. For example:
 ```
 File: hosts
 
-injector ansible_host=myinjectorhost.addr.com ansible_user=centos
+[hyperfoil_controller]
```

Comment:
Users do not have to know about hyperfoil. They do not even have to know hyperfoil is being used as test tool.

I would call this `injector_controller` and `injector_agent`.

And you should explain that multiple agents can be deployed if the injector host is the bottleneck.

Reply:
These names are used throughout the Hyperfoil Ansible integration scripts. Those can't be renamed for a downstream project. It means the naming of these 2 group vars will need to adopted.
 Certainly yes can explain that multiple agents can be used. A good suggestion.